### DISCUSSION TITLE
Sub Domínios

### DISCUSSION THREAD

**Discussion by BrenoAssis000dev** [balta-io/forum#196] [Tira Dúvidas] [answered]
Bom dia presados.
Fiquei com dúvida como aplicaria o Sub Domínio em um projeto, poderiam me dar um exemplo prático utilizando em um micro serviço e outro em um monólito?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BrenoAssis000dev, como vai?
Não entendi muito bem sua pergunta... consegue elaborar melhor o que ficou com dúvida?
